modules/ROOT/pages/getting-started.adoc

```diff
@@ -100,46 +136,22 @@ run and a given ECP which needs to pass for any Snapshot before that pipeline ca
+content is pushed to.
 
 === Release
-Every time you want to release newly built artifacts, you will create a Release CR in *your* namespace. The Release
+Every time you want to release newly built artifacts, you will create a Release CR in *your* tenant namespace. The Release
 CR represents your intent to release some content to customers. It is an active resource that, when present, will
 initiate the push of content.
 
-A Release CR references a specific Snapshot and ReleasePlan. It indicates the users' intention to ship the content
+A Release CR references a specific Snapshot and ReleasePlan. It indicates your intention to ship the content
 in the Snapshot by way of the referenced ReleasePlan.
 
 NOTE: It is possible to configure your ReleasePlan with xref:releasing:create-release-plan.adoc[auto-release].
 When enabled, the {ProductName} system will generate your Releases for you whenever your Snapshots pass all of their
-IntegrationTestScenarios. https://issues.redhat.com/browse/KONFLUX-1364[Future functionality], will allow you to
-automatically collect dynamic metadata for inclusion in the auto-generated Release CRs.
-
-== Working within Konflux
-
-All work will be performed in {ProductName} within a namespace. These can be "your" namespaces, team shared
-namespaces, or managed namespaces. If an action needs to be performed in a namespace, the work should generally
-be done with a Tekton PipelineRun. As you onboard Components to Konflux, you will be able to customize and define
-the PipelineDefinitions in order to build and test artifacts.
+IntegrationTestScenarios.
 
-=== Namespace vs Application vs Component vs Project/Product
-{ProductName} users operate in namespaces (with sole or shared ownership) which are scoped to individual projects
-(or project depending on the organization). One project can use multiple namespaces if desired, each with many
-applications with components. Namespaces should NOT be shared by many projects.
-
-=== What will I do in "my" namespaces
-
-You will create Components and Applications, run the PipelineRuns that are defined in your git repositories, view
-and iterate on the results of the IntegrationTestScenarios, and create Releases for specific Snapshots.
-
-=== What do I do with artifacts built from my namespaces
-
-Artifacts build in your namespace (from git pull/merge request or push events) will be pushed to an OCI registry
-along with their supporting metadata (including xref:metadata:index.adoc[SLSA provenance attestations and SBOMs]).
-Once the artifacts are pushed to the registry, they can be used for any common intended activity including development,
-testing, deployments, and releasing (i.e. pushing the images elsewhere either with credentials that you own or that
-someone else owns). The model of building artifacts in {ProductName} is to create them once and release them to as many
-places as needed/desired where each release action have a unique ECP gating the actions.
```

Comment:
Should we represent this somewhere else? It is an underlying concept that you can release anywhere as long as the policy gating that process succeeds.

Reply:
Not sure - but, let's leave it out for now.

Reply:
That's fine. I liked this addition, but I am biased as its author. I feel like this is one of the uniquely unintuitive parts about Konflux. It isn't just about building artifacts, but it is about building them and then doing something with them according to rules defined _out of your control_. A managed namespace is simply one that you don't control. It _could_ be considered a tenant namespace by some other set of users. But those users are the individuals that define the requirements for passing the namespace barrier successfully.